dChatServer/PlayerContainer.cpp

```diff
@@ -13,6 +13,50 @@
 #include "dConfig.h"
 #include "MessageType/Chat.h"
 
+#include "json.hpp"
+
+using json = nlohmann::json;
+
+void to_json(json& data, const PlayerData& playerData) {
+	data["id"] = playerData.playerID;
+	data["name"] = playerData.playerName;
+	data["gm_level"] = playerData.gmLevel;
+	data["muted"] = playerData.GetIsMuted();
+
+	auto& zoneID = data["zone_id"];
+	zoneID["map_id"] = std::to_string(playerData.zoneID.GetMapID());
```

Comment:
Why strings and not numbers? Is any of the too big for a f64?

Reply:
they physically cant be, all are <= uint32_t which fits in a f64.  not sure why they are strings tbh

Reply:
cause I just did it that way, will change